### ISSUE TITLE
Address fields in the Bank Information Form are confusing

### ISSUE THREAD

**Issue by kewitz** [opencollective/opencollective#3344]
We've been receiving support tickets asking if the address requested in the expense form is related to the beneficiary or the bank.

This is understandable since:
1. We already request the Physical Address right beside it;
2. In some countries, it is common to ask for the bank address.

### Our form

![image](https://user-images.githubusercontent.com/2119706/87698207-8a0fb480-c769-11ea-8cdc-00a13a879771.png)

### On TransferWise

In TransferWise docs, we find this topic about the recipient requirements. Notice this is slightly outdated since the requirements endpoint already returns address fields as required to most of their currency, including EUR.

> [...]
> 3) Address data Recipient address data is required only if target currency is USD, PHP, THB or TRY, or if the source currency is USD or AUD.
> 
> Country
> State (US, Canada, Brazil)
> City
> Address line
> Zip code
> When creating recipient, the following general rules should be applied to "accountHolderName" field:
> 
> Full names for personal recipients. They must include more than one name, and both first and last name must have more than one character.
> Business names must be in full, but can be just a single name. The full name cannot be just a single character but can be made up of a set of single characters. e.g. "A" is not permitted but "A 1" or "A1" is permitted.
> Digits are not allowed, except in business names.
> Special characters _()'*,. are allowed for personal and business names
> These requirements may vary depending of recipient type. A GBP example is provided here. You can find other currency examples below.
> As you can see many of the fields are null, in order to know which fields are required for which currency we expose the Recipients.Requirements endpoint.

This is how TransferWise form displays these fields, notice the tooltip:
![Screen record from 2020-07-16 13 45 08](https://user-images.githubusercontent.com/2119706/87699059-af50f280-c76a-11ea-8048-0d801fe3cc53.gif)

### Solution

I think we should keep it simple and implement something similar to the TransferWise form.

**Comment by znarf** [comment]
@kewitz Are we good that we're asking the address 2 times? Is there a way to handle that better?

**Comment by alanna** [comment]
I suggest this text for the tooltip:

> Address of the owner of the bank account (not the address of the bank)

@znarf 's comment points to the issue explained in #3340 - the address of the `pay to` profile and the payout method owner's address should always be the same, but in practice our system doesn't force this, and it probably should.

I suggest we implement the update to the text tooltip and follow up the bigger issue about addresses matching and redundant fields in the other issue.

**Comment by kewitz** [comment]
@znarf, I see two ways of simplifying this form:
1. Unifying by always requesting the structured address compatible with TransferWise.
    - Should require a bit of caching and requesting the backend to make sure the address is compatible with TW.
2. Hiding our free-form address field when the payout method is a wire transfer.
    - The quick solution.

There's also the fact that these addresses are stored in different places: the freeform address field is stored in the Expense while the Wire Transfer address is stored in the Payout Method. But the discussion about the use-cases and where we should persist the data overlaps with #3340.

**Comment by alanna** [comment]
If we hide the free form address field for bank transfer payments that already require an address field, please make sure the payee address is still printed on the expense summary screen and the downloadable invoice. We need payee addresses on the invoice in order for it to be valid. This also goes for when the payout method is Paypal - we need to collect and display the address for the invoice.
